web/src/components/PasswordWithMeter.test.js

```diff
@@ -2,28 +2,36 @@ import { shallow } from 'enzyme';
 import React from 'react';
 import sinon from 'sinon';
 
+// Import the lazy-loading zxcvbn so we can use jest's mocking functionality
+// eslint-disable-next-line no-unused-vars
+import zxcvbn from '../lazy/zxcvbn';
```

Comment:
The lazy-loading wrapper uses the `import()` function, which isn't actually supported by, like, anything yet, except Webpack.  Jest would reject a promise when `import()` was called, and that rejection wasn't handled anywhere so it bubbled up to an `unhandledRejection`.  It didn't interfere with our tests completing properly, but a future version of Node.js will crash on unhandled rejections.

Jest has built-in module mocking, though.  In your test, you import the module you want to mock, call `jest.mock(modulePath)`, and then when the thing being tested imports the module, Jest will give it the mock instead.  Handy dandy.

Reply:
I can't even tell you how much I love preemptive technical debt resolution. It's so beautiful :sob:

Reply:
In fairness, it made test output ugly because Node warned about it, and I was tired of ugly test output. 😆